src/global-messages/types.ts

```diff
@@ -0,0 +1,34 @@
+import {LanguageAndText} from '@atb/reference-data/types';
+import {Statuses} from '@atb/theme';
+
+export type AppPlatformType = 'ios' | 'android';
+
+export type GlobalMessageContext =
```

Comment:
I think it would be better if this is named `GlobalMessageContextType`. This would also make it less likely to be confused with the `GlobalMessagesContext`.